### PR TITLE
Implement opt-out

### PR DESCRIPTION
**Pull request by jmatraszek**
Didn't add anything to README.md, but I am not sure if that's needed. If yes, then I can do that.

### CODE REVIEW

**Comment by steveklabnik** [comment]
> Didn't add anything to README.md, but I am not sure if that's needed. If yes, then I can do that.

Doing so is nice, but not needed.

File: migrations/20170201210703_normalize_commits/down.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
+ALTER TABLE commits
+	ADD COLUMN author_name VARCHAR,
```
Comment (steveklabnik) [inline migrations/20170201210703_normalize_commits/down.sql]: When reviewing this, i missed that this was the down; I was very confused 😆

Comment (jmatraszek) [inline migrations/20170201210703_normalize_commits/down.sql]: :D

**Comment by jmatraszek** [comment]
Decided to add this to the README (however it starts to be a little bit overwhelming for a potential newcomer, so maybe there should be a concise README and a detailed USAGE) -- something to consider in the future. ;)

**Comment by steveklabnik** [comment]
Hey @jmatraszek ! So, so sorry this took so long.

I just committed https://github.com/rust-lang-nursery/thanks/commit/2002e0ae4c4a665b2d905b4e4ebac6a4d839aa8d, which means that we should be able to just merge stuff like this now, instead of needing my manual intervention. Woo!

Technically, since this does `ALTER TABLE`, it violates that rule, but I'm happy to just do it and if it fails for some reason fix things, so nbd. Merging, finally. Thanks so much for the patience.

**Comment by steveklabnik** [comment]
It appears to have deployed without issue! 🎊

**Comment by jmatraszek** [comment]
Cool! Thanks!

**Pull request merged**